### PR TITLE
if exist minified file, point to it

### PR DESCRIPTION
**Pull request by PeterDaveHello**
hope this is not too dirty to merge ...

### CODE REVIEW

File: lib/index.js [hunk 1]
```diff
@@ -203,8 +203,16 @@ CdnJsImporter.prototype.add = function (lib, callback) {
                         }
 
                         // Append the filename field used by cdnjs
-                        package.filename = files[0].replace(Path.join(repo.cwd, package.npmFileMap[0].basePath) + "/", "");
-
+                        pwd = Path.join(repo.cwd, package.npmFileMap[0].basePath) + "/";
+                        package.filename = files[0].replace(pwd, "");
+
+                        temp = package.filename.split(".");
```
Comment (IonicaBizau) [inline lib/index.js:211]: Please declare these variables with `var`, somewhere on line 172.

**Comment by IonicaBizau** [comment]
Thanks! Can you please add how this will change the behavior of the importer?

**Comment by PeterDaveHello** [comment]
@IonicaBizau updated, just make sure if there is minified file by the infix `.min`, if so, use it!

**Comment by IonicaBizau** [comment]
@PeterDaveHello Looks good! So, after this change, if there is a minified file, that will be used as main file, right? :four_leaf_clover:

**Comment by PeterDaveHello** [comment]
Yes!

**Comment by IonicaBizau** [comment]
I will merge this in `2.0.0`. I will also update the `is-there` dependency (#8) and add [CLP](https://github.com/IonicaBizau/node-clp) as arguments parser (#9).

**Comment by PeterDaveHello** [comment]
:+1:

**Comment by IonicaBizau** [comment]
@PeterDaveHello Hey, just noticed you also have a kitchen on GitHub! :fork_and_knife:

**Comment by PeterDaveHello** [comment]
Yep, hope GitHub can let me set it as default identify for me, ha, forked repo is too many to manage in the main id.

File: lib/index.js [hunk 2]
```diff
@@ -203,8 +206,15 @@ CdnJsImporter.prototype.add = function (lib, callback) {
...
-                        package.filename = files[0].replace(Path.join(repo.cwd, package.npmFileMap[0].basePath) + "/", "");
-
+                        package.filename = files[0].replace(pwd, "");
+
+                        temp = package.filename.split(".");
+                        ext = temp.pop();
+                        temp.push("min");
+                        temp.push(ext);
+                        if (IsThere.sync(pwd + temp.toString().replace(/\,/g, "."))) {
+                            package.filename = temp.toString().replace(/\,/g, ".");
```
Comment (IonicaBizau) [inline lib/index.js]: Am I missing something or you just want `temp.join(".")`?

Comment (PeterDaveHello) [inline lib/index.js]: lol, forgot what am I thinking then, thanks!

**Comment by PeterDaveHello** [comment]
hmmmm... any upadtes @IonicaBizau ?

**Comment by IonicaBizau** [comment]
@PeterDaveHello Oh, was busy with other projects, but thanks for ping. This week I will release [`git-stats 2.0.0`](https://github.com/IonicaBizau/git-stats/pull/47) and then I will take care of this project too. :smile: :four_leaf_clover:

**Comment by PeterDaveHello** [comment]
:+1:

**Pull request merged**